lib/apiservers/engine/backends/image.go

```diff
@@ -102,6 +114,8 @@ func (i *Image) ExportImage(names []string, outStream io.Writer) error {
 }
 
 func (i *Image) PullImage(ctx context.Context, ref reference.Named, metaHeaders map[string][]string, authConfig *types.AuthConfig, outStream io.Writer) error {
+	defer trace.End(trace.Begin("PullImage"))
```

Comment:
This will log the name of the fuction anyway. If supplying context, data from `ref` makes more sense.